docs/export-format.md

```diff
@@ -0,0 +1,158 @@
+# Briefcase export CSV format
+
+## Format parameters
+
+| Parameter               | Value | Description |
+| ----------------------- | ----- | ----------- |
+| Separator               | `,`   | Comma character |
+| Quoted                  | `"`   | Double quote character. See the [Values block columns](#values-block-columns) section to know which fields get quoted |
+| Header                  | Yes   | Every file will have a header with the respective column names |
+| Empty columns           | Yes   | See the "Values block columns" section to know which fields get empty columns |
+| Encoding of null values | No    | Nulls are just empty columns and, sometimes, empty strings |
+
+## Main submission output file
```

Comment:
Are you sure about this change? All throughout the code the concept that is used is "main" and "repeat" and parent might be misleading since a repeat group can be parent of a nested repeat group, and this second group can be also parent of a third nested group.